less/index.less

```
@@ -52,8 +52,155 @@ table {
 * {
     box-sizing: border-box;
 }
+body {
+    width:70%;
+    margin: 0 auto;}
+
+html {font-size: 63%;}
 
 html, body {
     height: 100%;
 }
 
+@small-font: 1.6rem;
+@medium-font: 2rem;
+@medium-large-font: 3.5rem;
+@large-font: 6rem;
+
+a, p, h4 {
+    font-size: @small-font;
+    line-height: 1.4;}
+
+h1 {font-size: @large-font;}
+
+h2 {font-size: @medium-large-font;}
+
+h3 {font-size: @medium-font;}
+
```

```suggestion
a, p, h4 {
    font-size: @small-font;
    line-height: 1.4;}

h1 {font-size: @large-font;}

h2 {font-size: @medium-large-font;}

h3 {font-size: @medium-font;}
```
Great job on using your variables here